.github/workflows/wpt-test.yml

```diff
@@ -0,0 +1,83 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "**/*.md"
+
+jobs:
+  test:
+    name: WebPageTest Test Cases
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: |
+          npm install webpagetest
+          npm install jest
+
+      - name: Run WebPageTest with unit tests
+        run: |
+          echo "::group::Unit tests"
+            npm test
+          echo "::endgroup::"
+        env:
+          WPT_SERVER: "webpagetest.httparchive.org"
```

Comment:
@pmeenan with new server we get an error:
```
console.error
      WPTAPIError {
        name: 'WPTAPIError',
        code: 301,
        message: 'Moved Permanently'
      }
```
Please adjust the configuration here.